apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -521,8 +521,8 @@ const BookingPage = ({
         <div
           className={classNames(
             "main",
-            isBackgroundTransparent ? "" : "dark:bg-darkgray-100 bg-default dark:border",
```

Comment:
dark mode classed needn't be applied after design token changes. It was pending to be fixed in this component.